libraries/botbuilder-dialogs-adaptive-runtime/src/coreBotAdapter.ts

```diff
@@ -1,85 +1,32 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { ClaimsIdentity, SkillValidation } from 'botframework-connector';
+import { BotFrameworkAuthentication } from 'botframework-connector';
+import { CloudAdapter, ConversationState, UserState, useBotState } from 'botbuilder';
 
-import {
-    ActivityEx,
-    BotFrameworkAdapter,
-    BotFrameworkAdapterSettings,
-    ConversationState,
-    InputHints,
-    MessageFactory,
-    TurnContext,
-    UserState,
-    useBotState,
-} from 'botbuilder';
-
-export class CoreBotAdapter extends BotFrameworkAdapter {
+export class CoreBotAdapter extends CloudAdapter {
     constructor(
-        settings: Partial<BotFrameworkAdapterSettings>,
+        botFrameworkAuthentication: BotFrameworkAuthentication,
         private readonly conversationState: ConversationState,
         userState: UserState
     ) {
-        super(settings);
+        super(botFrameworkAuthentication);
 
         useBotState(this, userState, conversationState);
 
-        this.onTurnError = async (turnContext, err) => {
+        this.onTurnError = async (context, err) => {
             console.error('[onTurnError] unhandled error', err);
 
-            await this.sendErrorMessage(turnContext, err);
-            await this.sendEoCToParentIfSkill(turnContext, err);
-            await this.clearConversationState(turnContext);
-        };
-    }
-
-    private async sendErrorMessage(turnContext: TurnContext, error: Error): Promise<void> {
-        try {
-            let errorMessageText = 'The bot encountered an error or bug';
-            let errorMessage = MessageFactory.text(errorMessageText, errorMessageText, InputHints.IgnoringInput);
-            await turnContext.sendActivity(errorMessage);
-
-            errorMessageText = 'To continue to run this bot, please fix the bot source code.';
-            errorMessage = MessageFactory.text(errorMessageText, errorMessageText, InputHints.ExpectingInput);
-            await turnContext.sendActivity(errorMessage);
-
-            await turnContext.sendTraceActivity(
-                'OnTurnError Trace',
-                error,
-                'https://www.botframework.com/schemas/error',
-                'TurnError'
-            );
-        } catch (err) {
-            console.error('Exception caught in sendErrorMessage', err);
-        }
-    }
-
-    private async sendEoCToParentIfSkill(turnContext: TurnContext, error: Error): Promise<void> {
```

Comment:
Note: handling removed as it didn't exist here: https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/CoreBotAdapter.cs

Reply:
Pushed to dialogHelper.ts run function.